### PR TITLE
[PR-17629] Accept phone number as a request parameter

### PR DESCRIPTION
**Pull request by hmlON**
Main PR - https://github.com/talkable/talkable/pull/8191

## Demo

https://void-docs.talkable.com/

## Related Stories

[](https://talkable.atlassian.net/browse/PR-17629)

### CODE REVIEW

File: source/advanced_features/params_encryption.rst [hunk 1]
```diff
@@ -131,6 +131,7 @@ Please modify the front-end using this pseudo code example:
    <script>
      _talkableq.push(['authenticate_customer', {
        email: '<%= to_json(TalkableEncryptionService.encrypt(current_user.email)) %>',
+       phone_number: '<%= to_json(current_user.phone_number) %>',
```
Comment (zhuravel) [inline source/advanced_features/params_encryption.rst:134]:
```suggestion
       phone_number: '<%= to_json(TalkableEncryptionService.encrypt(current_user.phone_number)) %>,
```

**Review by zhuravel — CHANGES_REQUESTED** [review]
Please update APIv2 sample responses and cURL examples.

**Review by zhuravel — APPROVED** [review]
You can add `phone_number` here:
https://github.com/talkable/talkable-docs/blob/d838a96dbc0fb6981e7556fcac9de6990f5f1d28/source/samples/ecommerce/platform/shopify_init_script.rst#L9-L13
```js
_talkableq.push(['authenticate_customer', {
  email:        '{{ customer.email }}',
  phone_number: '{{ customer.phone }}',
  first_name:   '{{ customer.first_name }}',
  last_name:    '{{ customer.last_name }}'
}]);
```

See `customer.phone` variable:
https://shopify.dev/api/liquid/objects/customer#customer-phone

**Comment by hmlON** [comment]
@zhuravel `customer.phone` is already [there](https://github.com/talkable/talkable-docs/blob/kholin-PR-17629/source/samples/ecommerce/platform/shopify_init_script.rst#L9-L13)

**Review by zhuravel — APPROVED** [review]

**Pull request merged**